scripts/plotting/test_training_distribution.py

```diff
@@ -0,0 +1,46 @@
+import matplotlib.pyplot as plt
+import xarray as xr
+
+def plot_label_distribution_split(data, split_indices, column='Label', bins=20, colors=['blue', 'green'], alpha=0.7, titles=['Training Set Label Distribution', 'Test Set Label Distribution'], x_label='Label', y_label='Frequency', figsize=(12, 6)):
```

Comment:
probably a code formatter should be introduced like [black](https://github.com/psf/black). In my project and at 4cast in general we use [pre-commit](https://pre-commit.com/) to handle these things for us. You could basically use my [pre-commit config file](https://gitlab.com/4cast1/maelstrom/a2/a2/-/blob/main/.pre-commit-config.yaml?ref_type=heads) and start from there.